pages/how-to-contribute.md

```diff
@@ -0,0 +1,151 @@
+---
+title: How To Contribute
+permalink: /how-to-contribute/
+redirect_from: /contributor/
+---
+
+Contributions are welcome to all Apache Accumulo repositories. While most contributions are code,
+there are other ways to contribute to Accumulo:
+
+* answer questions on mailing lists
+* review pull requests
+* verify and test new releases
+* update the Accumulo website and documentation
+
+Contributions are reviewed (via GitHub pull requests) by
+the community before being merged by a committer.
+
+This document provides basic instructions for contributing to Accumulo.  If you are looking for more information, check out the more comprehensive [contributor guide](/contributors-guide/).
+
+## Issues
+
+Any contribution should have a corresponding issue. Accumulo uses [JIRA] for issue tracking. Before creating an issue,
+you will need to create an [Apache JIRA account][jira-signup]. If you need help finding an issue to work on, check out
+the [open issues labeled for newbies][newbie-issues] or [contact us][contact].
+
+## Repositories
+
+Contributions can be made to the following repositories. While the general contribution workflow is
+described below, repositories have special instructions in their `CONTRIBUTING.md` file which can be
+viewed by clicking on `contribute` in the Links column below.
+
+| Repository                      | Links    | Description
+| ------------------------------- | -------- | -----------
+| [Accumulo][a]                   | [contribute][ac]  | Core Project
+| [Accumulo Website][w]           | [contribute][wc]  | Source for this website
+| [Accumulo Examples][e]          | [contribute][ec]  | Accumulo example code
+| [Accumulo Testing][t]           | [contribute][tc]  | Accumulo test suites such as continuous ingest and random walk
+| [Accumulo Docker][d]            | [contribute][dc]  | Source for Accumulo Docker image
+| [Accumulo Wikisearch][s]        | [contribute][sc]  | Accumulo example application that indexes and queries Wikipedia data
+
+## Contribution workflow
+
+1. Create an [Apache JIRA account][jira-signup] (for issue tracking) and [GitHub account][github-join] (for pull requests).
+1. Find an [issue][newbie-issues] to work on or create one that describes the work that you want to do.
+1. [Fork] and [clone] the GitHub repository that you want to contribute to.
+1. Create a branch in the local clone of your fork.
+```    
+    git checkout -b accumulo-4321
+```    
+1. Do work and commit to your branch. You can reference [this link][messages] for a guide on 
+to write good commit log messages in git.
```

Comment:
Need to correct `...for a guide on to write good commit...`

Reply:
@keith-turner, thanks for catching that.